java-frontend/src/test/java/org/sonar/java/UCFGJavaVisitorTest.java

```diff
@@ -151,6 +151,26 @@ public void build_parameter_annotations() {
     assertCodeToUCfg("class A { String method(@javax.annotation.Nullable @org.springframework.web.bind.annotation.RequestParam() @org.springframework.format.annotation.DateTimeFormat String arg) { return \"foo\";}}", expectedUCFG);
   }
 
+  @Test
+  public void unknown_method() {
+    Expression.Variable arg = UCFGBuilder.variableWithId("arg");
+
+    UCFG expectedUCFG = UCFGBuilder.createUCFGForMethod("A#method(Ljava/util/Set;)V").addMethodParam(arg)
+      .addBasicBlock(newBasicBlock("0")
+        .ret(constant("implicit return"), new LocationInFile(FILE_KEY, 9, 2, 9, 3)))
+      .build();
+    assertCodeToUCfg("import java.util.Set;\n" +
+      "import java.util.Collection;\n" +
+      "import java.util.stream.Collectors;\n" +
+      "public class A {\n" +
+      "  void method(Set<String> arg) { \n" +
+      "    arg.stream()\n" +
+      "      .flatMap(Collection::stream)\n" +
+      "      .collect(Collectors.toCollection(LinkedHashSet::new)); \n" +
+      "  }\n" +
+      "}", expectedUCFG);
```

Comment:
This is unreliable test : that might be fix by improvement of semantic and so make this unrelated test fail. 
A nicest way to test that is just to invoke a method named "__unknown" which is not defined in your source.